ts/features/fci/store/reducers/fciPollFilledDocument.ts

```diff
@@ -0,0 +1,64 @@
+import { getType } from "typesafe-actions";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import { createSelector } from "reselect";
+import { Action } from "../../../../store/actions/types";
+import { fciClearStateRequest, fciPollFilledDocument } from "../actions";
+import { GlobalState } from "../../../../store/reducers/types";
+import { NetworkError } from "../../../../utils/errors";
+
+export const POLLING_FREQ_TIMEOUT = 2000 as Millisecond;
+export const MAX_POLLING_RETRY = 5;
```

Comment:
@grausof Are these values acceptable?

Reply:
what is the unit of measurement? I recommend putting it in the variable name!

Reply:
As you can see the `POLLING_FREQ_TIMEOUT` is in Millisecond. The `MAX_POLLING_RETRY` is the number of times the saga try to fetch the URL. Feel free to change the const name with one more explanatory.

Reply:
ok thanks, I think they are just fine!